core-v1/src/main/scala/debug/DebugArgs.scala

```diff
@@ -183,9 +183,9 @@ object DebugArgs {
     A,
     B,
     OP[_],
-  ]: Aux[Expr.Exists[C, A, B, OP], OP, (C[A], Either[List[B], NonEmptyList[B]]), B] =
+  ]: Aux[Expr.Exists[C, A, B, OP], OP, (C[A], Either[Vector[B], NonEmptyVector[B]]), B] =
```

Comment:
I think I should use `Seq` and `NonEmptySeq` here to hide implementation details, in case another data type would be more performant...